assignments/array-methods.js

```diff
@@ -458,11 +458,19 @@ const runners = [
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+runners.forEach(function (runner) {
+  fullName.push(`${runner.first_name} ${runner.last_name}`);
+})
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
-let allCaps = [];
+// let allCaps = [];
+runners.map(function (runner) {
+  I $ { ` dont know how to do this stuff at all `} + ;asdjfhoiausflhjoiqwefuihalsfhoihasfiuh
+}
+
+
```

Comment:
allcaps = runners.map(function(runner) {
return ... something
}
inside is going to be runner. something . something
then console.log